blockchains/xrp/xrp_worker.js

```diff
@@ -10,6 +10,8 @@ class XRPWorker extends BaseWorker {
     super();
     this.nodeURLs = constants.XRP_NODE_URLS.split(',');
     this.connections = [];
+    this.emptyTransactionHash = '0000000000000000000000000000000000000000000000000000000000000000';
```

Comment:
i'd prefer something like '0'.repeat(160) or something like this. Let PC do hard work for you ;)